_includes/badge-generator.html

```diff
@@ -43,8 +43,8 @@
     var version = $("#version-sel").val();
     var style = $("#style-sel").val();
     $("#shield").attr("src", "/assets/badges/scalajs-"+version+".svg");
-    var code = style == "Markdown" ? "[![Scala.js](http://scala-js.org/assets/badges/scalajs-"+version+".svg)](http://scala-js.org)" :
-      '<img src="http://scala-js.org/assets/badges/scalajs-'+version+'.svg">';
+    var code = style == "Markdown" ? "[![Scala.js](https://scala-js.org/assets/badges/scalajs-"+version+".svg)](https://www.scala-js.org)" :
+      '<img src="https://scala-js.org/assets/badges/scalajs-'+version+'.svg">';

```

Comment:
Yeah, adding www is good. But `production_url` doesn't give the HTTPS we want, so that there are never any security conflicts in sites using the badges. Better to hardcode the HTTPS there in this case.